packages/jtex/src/export.ts

```diff
@@ -0,0 +1,4 @@
+export function pdfExportCommand(texFile: string, logFile: string, engine?: string) {
+  if (!engine) engine = 'xelatex';
```

Comment:
Just making sure you know about this syntax @fwkoch:
```suggestion
export function pdfExportCommand(texFile: string, logFile: string, engine = 'xelatex') {
```